packages/@okta/vuepress-site/docs/guides/build-custom-ui-mobile/main/index.md

```diff
@@ -0,0 +1,76 @@
+---
+title: Build a custom sign-in UI in your mobile app
+excerpt: Learn how to build a custom sign-in UI in your mobile app.
+layout: Guides
+---
+
+You can connect your mobile app to Okta and sign users in by [opening a browser](/docs/guides/sign-into-mobile-app/before-you-begin/), however, if you prefer that your users not leave your app, you need to build a custom sign-in UI with native controls and screens instead. Use this guide to build a customized sign-in experience inside your mobile application.
```

Comment:
Yeah, this was a language neutral link. I've "fixed" it to what will be correct by the time these PRs are all merged (it would currently be broken). I'm not sure if this is going to work, but we will see what happens when I try it.